app/build.gradle

```diff
@@ -1,14 +1,15 @@
 apply plugin: 'com.android.application'
```

Comment:
Should create a separate branch for kotlin + Databinding + AndroidX improvements!